WORKLIST.txt

```diff
@@ -108,6 +109,9 @@ STABILITY:
 - 12.14.2017 - Just saw Star Wars 8 tonight, woot. Also, JMoria beats the heck out of the CPU, I left it running, and it was using 90%+ CPU, caused the fan to come on... maybe if it was locked to a particular framerate (like 30fps), it would give the CPU more of a break in between cycles?
 
 
+DUNGEON:
+- Should rooms further away from the stairs have tougher / more monsters? Better item drops?
```

Comment:
moria is built vertically; monsters only spawn at appropriate depths, the thing you're asking for is "should deeper levels contain harder monsters" and the answer is "it's already there".